src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/ValuationInformationPolicyServiceImplTests.java

```diff
@@ -0,0 +1,153 @@
+                .thenReturn(accountingPoliciesApi);
+        doNothing().when(accountingPoliciesTransformer)
+                .setValuationInformationPolicy(valuationInformationPolicy, accountingPoliciesApi);
+        when(accountingPoliciesService.updateAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, accountingPoliciesApi))
+                .thenReturn(apiErrors);
+
+        List<ValidationError> validationErrors =
+                service.submitValuationInformationPolicy(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, valuationInformationPolicy);
+
+        assertEquals(apiErrors, validationErrors);
+
+        verify(accountingPoliciesService, times(1))
+                .getAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID);
+
+        verify(accountingPoliciesTransformer, times(1))
+                .setValuationInformationPolicy(valuationInformationPolicy, accountingPoliciesApi);
+
+        verify(accountingPoliciesService, times(1))
+                .updateAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, accountingPoliciesApi);
+    }
+
+    @Test
+    @DisplayName("Submit provided valuation information policy")
+    void submitProvidedValuationInformationPolicy() throws ServiceException {
+
+        ValuationInformationPolicy valuationInformationPolicy = new ValuationInformationPolicy();
+        valuationInformationPolicy.setIncludeValuationInformationPolicy(true);
+        valuationInformationPolicy.setValuationInformationPolicyDetails("valuationInformationPolicy");
+
+        AccountingPoliciesApi accountingPoliciesApi = new AccountingPoliciesApi();
+        when(accountingPoliciesService.getAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID))
+                .thenReturn(accountingPoliciesApi);
+        doNothing().when(accountingPoliciesTransformer)
+                .setValuationInformationPolicy(valuationInformationPolicy, accountingPoliciesApi);
+        when(accountingPoliciesService.updateAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, accountingPoliciesApi))
+                .thenReturn(apiErrors);
+
+        List<ValidationError> validationErrors =
+                service.submitValuationInformationPolicy(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, valuationInformationPolicy);
+
+        assertEquals(apiErrors, validationErrors);
+
+        verify(accountingPoliciesService, times(1))
+                .getAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID);
+
+        verify(accountingPoliciesTransformer, times(1))
+                .setValuationInformationPolicy(valuationInformationPolicy, accountingPoliciesApi);
+
+        verify(accountingPoliciesService, times(1))
+                .updateAccountingPoliciesApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, accountingPoliciesApi);
+    }
+
+    /**
+     * Creates a valuation information policy with 'include policy' set to true, but without policy details
+     * @return
+     */
```

Comment:
Don't need the Java Doc here as your method name explains what's occurring. As per previous discussions Java Docs are not required on private methods in these situations.